File: fizzbuzz.cpp
#include <stdio.h>
 
int main(void)
{
    for (int i=1; i<=100; i++)
        printf(i%15==0?"FizzBuzz\t": i%3==0? "Fizz\t" : i%5==0? "Buzz\t" : "%d\t", i);
        
    return 0;
}

File: fizzbuzz.py
'''for i in range(1,101):
    print("Fizz"*(i%3<1)+(i%5<1)*"Buzz" or i)'''
    
    
print(list("Fizz"*(i%3<1)+"Buzz"*(i%5<1) or i for i in range(1,100)))